armv7hf-buildpack-deps/README-short.txt

```diff
@@ -1 +1 @@
-A collection of common build dependencies used for installing various modules, e.g., gems.
+This repository is obsolete.
```

Comment:
we don't have buildpack-deps image for each arch, we have for each device type. So not sure where to point to